src/tests/ftest/container/multiple_delete.py

```diff
@@ -53,7 +53,7 @@ def test_multiple_container_delete(self):
             self.run_ior_with_pool(create_cont=False)
             self.container.destroy()
             scm_fs, ssd_fs = self.get_pool_space()
-            out.append("iter = {}, scm = {}, ssd = {}".format(i+1, scm_fs, ssd_fs))
+            out.append("iter = {}, scm = {}, ssd = {}".format(loop+1, scm_fs, ssd_fs))
```

Comment:
Only if you need to push again - This warning shows up silently in flake, but we should have spaces around binary operators
```suggestion
            out.append("iter = {}, scm = {}, ssd = {}".format(loop + 1, scm_fs, ssd_fs))
```

Reply:
Fixed